### PR TITLE
scHiCExplorer version 1

### PR DESCRIPTION
**Pull request by joachimwolff**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

@bgruening Galaxy wrappers for the scHiCExplorer (single-cell HiCExplorer).

### CODE REVIEW

**Comment by joachimwolff** [comment]
@bgruening They pass now.

File: tools/schicexplorer/scHicDemultiplex.xml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    <expand macro="requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+        #import re
+        mkdir outputFiles &&
+        #set fastqFile = @ESCAPE_IDENTIFIER_FASTQ@
+        ln -s '$fastq' '$fastqFile' &&
+        @BINARY@
+
+        --fastq $fastqFile
+        --barcodeFile $barcodeFile
```
Comment (mvdbeek) [inline tools/schicexplorer/scHicDemultiplex.xml]:
```suggestion
        --barcodeFile '$barcodeFile'
```

File: tools/schicexplorer/scHicDemultiplex.xml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        --srrToSampleFile $srrToSample
```
Comment (mvdbeek) [inline tools/schicexplorer/scHicDemultiplex.xml]:
```suggestion
        --srrToSampleFile '$srrToSample'
```

File: tools/schicexplorer/scHicMergeMatrixBins.xml [hunk 3]
```diff
@@ -0,0 +1,367 @@
...
+    <description>change the resolution of the scHi-C matrices</description>
+    <macros>
+        <token name="@BINARY@">scHicMergeMatrixBins</token>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+        @BINARY@
+
+        --matrix $matrix_mcooler
```
Comment (mvdbeek) [inline tools/schicexplorer/scHicMergeMatrixBins.xml]:
```suggestion
        --matrix '$matrix_mcooler'
```

File: tools/schicexplorer/scHicNormalize.xml [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+    <description>normalize single-cell Hi-C interaction matrices to the same read coverage</description>
+    <macros>
+        <token name="@BINARY@">scHicNormalize</token>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+        @BINARY@
+
+        --matrix $matrix_mcooler
```
Comment (mvdbeek) [inline tools/schicexplorer/scHicNormalize.xml]:
```suggestion
        --matrix '$matrix_mcooler'
```

File: tools/schicexplorer/scHicQualityControl.xml [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+    <description>quality control for single-cell Hi-C interaction matrices</description>
+    <macros>
+        <token name="@BINARY@">scHicQualityControl</token>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+        @BINARY@
+
+        --matrix $matrix_mcooler
```
Comment (mvdbeek) [inline tools/schicexplorer/scHicQualityControl.xml]:
```suggestion
        --matrix '$matrix_mcooler'
```

**Comment by bgruening** [comment]
Great thanks!

**Pull request merged**